spark/src/test/scala/org/apache/spark/sql/comet/CometPlanStabilitySuite.scala

```diff
@@ -262,6 +262,7 @@ trait CometPlanStabilitySuite extends DisableAdaptiveExecutionSuite with TPCDSBa
       CometConf.COMET_EXEC_ENABLED.key -> "true",
       CometConf.COMET_DPP_FALLBACK_ENABLED.key -> "false",
       CometConf.COMET_EXEC_SHUFFLE_ENABLED.key -> "true",
+      CometConf.COMET_EXEC_SORT_MERGE_JOIN_WITH_JOIN_FILTER_ENABLED.key -> "true",
```

Comment:
@andygrove Do you remember by any chance why `COMET_EXEC_SORT_MERGE_JOIN_WITH_JOIN_FILTER_ENABLED` is enabled only for `testQuery()` but not for `createSparkSession()`?

Reply:
Maybe this one? https://github.com/apache/datafusion-comet/issues/901